config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -625,3 +625,56 @@ periodics:
     testgrid-tab-name: sig-network-kind, multizone
     description: Runs tests against a Multizone Kubernetes in Docker cluster
     testgrid-alert-email: antonio.ojea.garcia@gmail.com, robertjscott@google.com
+# network test against kubernetes master branch with `kind` and kube-proxy detectLocalMode="NodeCIDR"
+- interval: 6h
+  name: ci-kubernetes-kind-network-detect-local-node-cidr
+  cluster: k8s-infra-prow-build
+  labels:
+    preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  decorate: true
+  decoration_config:
+    timeout: 200m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+    path_alias: "k8s.io/test-infra"
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/krte:v20220221-c13e827224-master
+      command:
+        - wrapper.sh
+        - bash
+        - -c
+        - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && $GOPATH/src/k8s.io/test-infra/experiment/kind-detect-local-e2e.sh
```

Comment:
Use experiment/kind-detect-local-e2e.sh instead of the "hack/ci/e2e-k8s.sh" script in the kind repo.